src/data.ts

```diff
@@ -43,6 +43,6 @@ export const setupSession = (set: Setter) => {
   })
 }
 
-export const authenticateUser = (provider: Provider) => client.auth.signIn({ provider })
+export const authenticateUser = async (provider: Provider) => client.auth.signIn({ provider })
```

Comment:
what's putting the async here?

Reply:
`@typescript-eslint/promise-function-async` -> https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/promise-function-async.md

TL;DR if a function returns a Promise the `() => Promise<Result>` should be `async () => Result`

Reply:
I'm not a fan of adding extraneous `async` directives. It doesn't seem to be a default rule, can we get rid of it?

Reply:
sure thing.